pages/linux/smem.md

```diff
@@ -0,0 +1,20 @@
+# smem
+
+> Show memory usage for programs.
+> More information: <https://manned.org/smem>.
+
+- Print memory usage for current processes:
+
+`smem`
+
+- Print memory usage for current processes for a every user on a system:
```

Comment:
```suggestion
- Print memory usage for all current processes for an every user on a system:
```

Reply:
I think adding `a/an` does not sound right. Although, not an expert but in my opinion, I guess it should be:
```suggestion
- Print memory usage for all current processes for every user on the system:
```

Reply:
@CleanMachine1 wdyt about the suggestion of mfrw?

Reply:
I dont think it is all current processes therefore don't think these changes are necessary, to be honest I don't know much about the command.

Reply:
ok, let's merge then.